packages/p2p/src/types/my-ads.types.ts

```diff
@@ -0,0 +1,85 @@
+export type TPaymentMethodOrder = {
+    bank_transfer?: number;
+    other?: number;
+    wechat_pay?: number;
+    alipay?: number;
+    skrill?: number;
+    paypal?: number;
+};
+
+export type TPaymentMethodField = {
+    display_name: string;
+    required: number;
+    type: string;
+    value?: string;
+};
+
+export type TPaymentMethod = {
+    ID: string;
+    name: string;
+    is_enabled: number;
+    method: keyof TPaymentMethodOrder;
+    display_name: string;
+    fields: {
+        account: TPaymentMethodField;
+        instructions: TPaymentMethodField;
+    };
+};
+
+type TAdvertiserDetails = {
+    completed_orders_count: number;
+    first_name?: string;
+    id: string;
+    is_online: number;
```

Comment:
Isn't this `boolean`?

```suggestion
    is_online: boolean;
```

Reply:
its actually 0 and 1. since at the other places its typed as number, i followed the same for these as well. changing the type here will require changing it at all the other places as well.

Reply:
```suggestion
    is_online: 1 | 0;
```